paddle/fluid/framework/new_executor/interpretercore_util.cc

```diff
@@ -370,6 +364,12 @@ void build_op_func_list(const platform::Place& place,
 
       op_func_node.kernel_func_ = OpKernelComputeFunc(kernel_iter->second);
       op_func_node.kernel_func_(exec_ctx);
+
+      // post-process grad_op.outputs if need cast complex grad into real grad.
+      // NOTE(Aurelius84): insert a transfer_dtype_op inplacely to cast it.
```

Comment:
Is `if framework::IsComplexType(kernel_type_->data_type_)`(in opereator.cc) needed?

Reply:
Thanks, done